tests/module_user.py

```diff
@@ -8,11 +8,7 @@
 
 class ModuleUserTestCase(unittest.TestCase):
```

Comment:
Seems like this class is not needed anymore?

Reply:
I guess so. I was not sure if it should be deleted (I thought maybe you have some plans for this).

Reply:
Nope, feel free to remove so.

Reply:
Done